crates/components/src/sound_tab.rs

```diff
@@ -22,57 +22,70 @@
 // terms of the Steamworks API by Valve Corporation, the licensors of this
 // Program grant you additional permission to convey the resulting work.
 
+use crate::UiExt;
+
 pub struct SoundTab {
     /// The source for this tab.
     pub source: luminol_audio::Source,
-    pub volume: u8,
-    pub pitch: u8,
-    pub selected_track: String,
+    pub audio_file: luminol_data::rpg::AudioFile,
+
+    search_text: String,
     folder_children: Vec<luminol_filesystem::DirEntry>,
+    filtered_children: Vec<luminol_filesystem::DirEntry>,
 }
 
 impl SoundTab {
     /// Create a new SoundTab
     pub fn new(
         filesystem: &impl luminol_filesystem::FileSystem,
         source: luminol_audio::Source,
+        audio_file: luminol_data::rpg::AudioFile,
     ) -> Self {
-        let folder_children = filesystem.read_dir(format!("Audio/{source}")).unwrap();
+        let mut folder_children = filesystem.read_dir(format!("Audio/{source}")).unwrap();
+        folder_children.sort_unstable_by(|a, b| a.file_name().cmp(b.file_name()));
         Self {
             source,
-            volume: 100,
-            pitch: 100,
-            selected_track: String::new(),
+            audio_file,
+
+            filtered_children: folder_children.clone(),
+            search_text: String::new(),
             folder_children,
         }
     }
```

Comment:
The volume and pitch should default to 100 like before. The volume defaults to 0 and the pitch defaults to 50 now.